presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/ExpressionVerifier.java

```diff
@@ -71,6 +70,16 @@ protected Boolean visitNode(Node node, Expression context)
     }
 
     @Override
+    protected Boolean visitInPredicate(InPredicate actual, Expression expectedExpression)
```

Comment:
These new methods should go in a separate commit (vs the one that adds support for matching apply in plan assertion dsl)